apps/datahub/src/app/record/record-otherlinks/record-otherlinks.component.html

```diff
@@ -1,37 +1,11 @@
-<div class="flex flex-row gap-x-4 items-center justify-center md:justify-start">
```

Comment:
keeping the title in the app is better IMO because the layout is typically the responsibility of the app; was there a reason to put in back in the layout lib?